pageserver/src/tenant.rs

```diff
@@ -1505,7 +1505,7 @@ impl Tenant {
                     // activated should never be marked as broken. We cope with it the best
                     // we can, but it shouldn't happen.
                     *current_state = TenantState::Broken;
-                    warn!("Changing Active tenant to Broken state");
+                    warn!("Changing Active tenant to Broken state, reason:{}", reason);
```

Comment:
Will it be better to add space after column? "Such:message" IMHO is harder to read "than: such".